common/testutils/db.go

```diff
@@ -0,0 +1,32 @@
+package testutils
+
+/*
+	There is a mock packge, https://github.com/DATA-DOG/go-sqlmock, that could be used for these tests
+	instead of a real mysql database. However, testing against a real mysql database better suits
+	our needs as we focus more heavily on integration testing rather than unit testing.
+*/
+
+import (
+	"flag"
+	"fmt"
+	"github.com/quintilesims/layer0/common/db"
+)
+
+var (
+	username = flag.String("username", "layer0", "username for the test db")
+	password = flag.String("password", "nohaxplz", "password for the test db")
```

Comment:
What are you thinking? I don't want to use something like `root` since that comes baked into mysql.